tripal/src/Form/TripalEntitySettingsForm.php

```diff
@@ -139,18 +157,23 @@ public function validateForm(array &$form, FormStateInterface $form_state) {
   public function submitForm(array &$form, FormStateInterface $form_state) {
     $allowed_title_tags = $form_state->getValue('allowed_title_tags');
     $drupal_entity_field_store = $form_state->getValue('default_cache_backend_field_values');
-    // Trim, convert to lower case, and collapse multiple spaces for consistency.
+    // Trim, convert to lower case, and collapse multiple spaces
+    // for consistency.
     $allowed_title_tags = strtolower(trim($allowed_title_tags));
     $allowed_title_tags = preg_replace('/ +/', ' ', $allowed_title_tags);
 
     $widget_global_select_limit = trim($form_state->getValue('widget_global_select_limit'));
+    $publish_global_max_delta = trim($form_state->getValue('publish_global_max_delta'));
+    $publish_global_max_delta_inhibit = $form_state->getValue('publish_global_max_delta_inhibit');
 
-    // Update configuration
+    // Update configuration.
     \Drupal::configFactory()
       ->getEditable('tripal.settings')
```

Comment:
The config factory is available in all forms via dependency injection! See https://api.drupal.org/api/drupal/core%21lib%21Drupal%21Core%21Form%21FormBase.php/property/FormBase%3A%3AconfigFactory/11.x

That means this can be

```suggestion
    $this->configFactory->getEditable('tripal.settings')
```